spec/factories/poll_factory.rb

```diff
@@ -13,6 +13,10 @@
       title ''
     end
 
+    trait :with_ready_status do
```

Comment:
Is this a really valuable factory trait?
Just `create :poll, status: :ready` will do the same, while being shorter.